### PR TITLE
Address PR feedback from pdehaan

### PR DESCRIPTION
**Pull request by pdehaan**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,39 +8,39 @@ ARG TEST_CONFIG="{'error':'missing_config'}"
...
+    apt-get install -y -qq software-properties-common && \
     apt-get install -y -qq wget unzip \ 
                        libreadline-gplv2-dev libncursesw5-dev \
                        libssl-dev libsqlite3-dev tk-dev libgdbm-dev \
                        libc6-dev libbz2-dev libgtk-3-0 libdbus-glib-1-2 \
-                       libgtk-3-dev libglib2.0-dev &&\
-                       rm -rf /var/lib/apt/lists/* 
+                       libgtk-3-dev libglib2.0-dev && \
+    rm -rf /var/lib/apt/lists/* && \
+    apt-get clean
```
Comment (pdehaan) [inline Dockerfile]: Per https://linux.die.net/man/8/apt-get:

> - **clean:** Clears out the local repository of retrieved package files. It removes everything but the lock file from /var/cache/apt/archives/ and /var/cache/apt/archives/partial/.
> - **autoclean:** Like clean, autoclean clears out the local repository of retrieved package files. The difference is that it only removes package files that can no longer be downloaded, and are largely useless. This allows a cache to be maintained over a long period of time without it growing out of control. The configuration option APT::Clean-Installed will prevent installed packages from being erased if it is set to off.

File: Dockerfile [hunk 2]
```diff
@@ -8,39 +8,39 @@ ARG TEST_CONFIG="{'error':'missing_config'}"
 ENV MOZ_HEADLESS="1"
 
 RUN apt-get -y update && \
-    apt-get install -y -qq software-properties-common &&\
+    apt-get install -y -qq software-properties-common && \
```
Comment (pdehaan) [inline Dockerfile]: Added a space before the `\` in a few places below for consistency.

File: stage-config.json.asc [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
+KsrWKIwjA7hm4L/6O7wm8mP2DsPPR8d3c/OshWmPJAvtESf56vVGkejuFs15L0Dl
+oQypxzxptkmEX5E5KIYoIuDjXOE3sfdGPGLmPAZKf1fGv3P3ORF5/e2z1MG2xynK
+oWEkzgUyDn1PpbXk2xdP4pXGXhS8ZTfYXay6yIq9OTdVYoQMZVyWyPnNGpLsFBA9
+JOkdmUzyfk4ZzmubDk3TynbAD7YiywratOSMf6rsTE0mUnMI3zHefhIWH0YGIF6l
+SXiK3Wkn1z0EhNsZ78XXep/ARx2hWIHo8cETIlBflwlA975Z+0jrDYJwV5SHFk+9
+FhHLpPuTrbM1oRkLtHyO+SUk6Gjm6GOugTN2H5BrL4qywE7QcdtFOXGmAIHtyXSa
+s1YyC+KMSCYwR5fCE5yxkZ6Ahv776rLvxwVkRHkUj/dI5aF9u+oxBeTwgiXUT9kp
+h5WvFNbU+RETLhIWOMVXlwN6Uutdl6QGSyvPWQ+Suw0sow==
+=gxYj
+-----END PGP MESSAGE-----
```
Comment (pdehaan) [inline stage-config.json.asc]: These files were changed to match what is currently in the upstream/master: https://github.com/mozilla-services/sync-tps-setup

**Comment by rpappalax** [comment]
awesome @pdehaan 👍

**Pull request merged**